docs/pages/choose-an-edition/teleport-cloud/faq.mdx

```diff
@@ -164,6 +168,18 @@ connected agents?](#how-can-i-find-version-information-on-connected-agents).
 For more information about automatic updates and compatibility issues, contact
 [Teleport support](https://goteleport.com/support/).
 
+### Why wasn't my Teleport Enterprise Cloud instance upgraded to the latest major version?
+
+The Teleport Control plane supports agents that are on the same major version or one major version behind.
+If your agents are not up to date, we will withhold major version upgrades in order to prevent connectivity
```

Comment:
We're standardizing the docs on capitalizing "Teleport Agent" since it's a Teleport-specific component.